src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="top-bar">
+      <a href="#" class="top-bar__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="img_logo"
+        >
+      </a>
+
+      <nav class="top-bar__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link "
+               data-qa="nav-hover">laptops & computers</a>
```

Comment:
If tag has more then 2 attributes - they should go on new line
```suggestion
            <a href="#" class="nav__link "
               data-qa="nav-hover">laptops & computers</a>
```
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="nav-hover"
            >
              laptops & computers
            </a>
```

Reply:
https://github.com/mate-academy/layout_catalog/pull/3437/files#r1277475540
didn't fix it